how-to-use-azureml/machine-learning-pipelines/intro-to-pipelines/aml-pipelines-data-transfer.ipynb

```diff
@@ -119,15 +119,16 @@
       "source": [
         "from msrest.exceptions import HttpOperationError\n",
         "\n",
-        "blob_datastore_name='MyBlobDatastore'\n",
-        "account_name=os.getenv(\"BLOB_ACCOUNTNAME_62\", \"<my-account-name>\") # Storage account name\n",
-        "container_name=os.getenv(\"BLOB_CONTAINER_62\", \"<my-container-name>\") # Name of Azure blob container\n",
-        "account_key=os.getenv(\"BLOB_ACCOUNT_KEY_62\", \"<my-account-key>\") # Storage account key\n",
         "\n",
         "try:\n",
         "    blob_datastore = Datastore.get(ws, blob_datastore_name)\n",
         "    print(\"found blob datastore with name: %s\" % blob_datastore_name)\n",
         "except HttpOperationError:\n",
+        "        blob_datastore_name='MyBlobDatastore'\n",
```

Comment:
mistake?

Reply:
Yes, my bad!

Reply:
Seems it's still wrong, does this file need to be changed at all?

Reply:
You are right.
I was thinking to place in except block the os.getenv since if the datastore exists there is no need to retrieve this info.
So, besides fixing the indentation, this should belong to a different pull request.
Do you agree?